### DISCUSSION TITLE
profile gros_trail

### DISCUSSION THREAD

**Discussion by fixifoxy** [OsmAnd-Rendering/Motorcycle#10] [General]
Je suppose que Christoph  et synonym a cri-cri
J utilise ton profile gros-trail .
et j ai des problems avec la fonction "create  route" with existing track. Cela a ete discuter on Github Osmand  closed issue
#  17045 . Et Osmands me disent que ton profile "is broken". Qu est-ce que je peut faire?
HOrst d Autriche

**Comment by cricri-du-lauragais** [comment]
depuis la version 4.4 les options ont changées de place,
maintenant c'est pour Android

ouverture du gpx dans osmand
clic sur la trace affichée sur la carte
clic sur l'icone du crayon (modifier la trace)
options
déplacer sur les routes
choisir parmi les routages proposés
